js/src/utilities/notify-content-updated.ts

```diff
@@ -0,0 +1,6 @@
+/**
```

Comment:
Do we need this?

Reply:
I brought it here to safely replace dismiss in docs-ui. `notifyContentUpdated` called in dismissing code.

Reply:
I'm not sure this is something Atlas should do, two things:
- do we want to introduce a possible naming conflict with `notifyContentUpdated`?
- do we want to prefix `content-update` custom event name with something?

Reply:
Refactored this section, removed completely `notifyContentUpdated`, and added `dismiss-content-update` custom even directly in dismiss code.